src/task2/router/users.js

```diff
@@ -3,12 +3,12 @@ const { addUser, retrieveUser, updateUser, deleteUser, getAutoSuggestUsers } = r
 const router = express.Router();
 
 router.route('/')
-    .get((req, res) => getAutoSuggestUsers(req, res))
-    .post((req, res) => addUser(req, res));
+    .get((req, res, next) => getAutoSuggestUsers(req, res, next))
```

Comment:
you can simplify it just to 
```
.get(getAutoSuggestUsers)
.post(addUser)
```

the same redundant code could be simplified for every route